sympy/parsing/latex.py

```diff
@@ -0,0 +1,545 @@
+# Ported from latex2sympy by @augustt198
+# https://github.com/augustt198/latex2sympy
```

Comment:
You'll need to include their license or better yet getting their permission to license this under the SymPy license is best.

Reply:
Added in LICENSE.txt for now!

@augustt198, can you provide any feedback regarding licensing not covered by your initial response to augustt198/latex2sympy#1?